src/cozmo/world.py

```diff
@@ -926,6 +931,7 @@ def _remove_fixed_custom_object_instances(self):
         Returns:
             A :class:`cozmo.objects.FixedCustomObject` instance with the specified dimensions and pose.
         '''
+        #pylint: disable=no-member
```

Comment:
does pylint require that these are at the top of the file, or can they be placed next to the problem line - would make it easier to see what line is causing the issue.